packages/alea-frontend/pages/exp/gpt-templates.tsx

```diff
@@ -0,0 +1,172 @@
+import CloseIcon from '@mui/icons-material/Close';
+import { Box, Button, IconButton, Typography } from '@mui/material';
+import { Template, getTemplateVersions, getTemplates } from '@stex-react/api';
+import {
+  FixedPositionMenu,
+  LayoutWithFixedMenu,
+} from '@stex-react/stex-react-renderer';
+import { PRIMARY_COL, shouldUseDrawer } from '@stex-react/utils';
+import { NextPage } from 'next';
+import { useEffect, useState } from 'react';
+import MainLayout from '../../layouts/MainLayout';
+import { KeyValueDisplay } from './gpt-eval';
+import { GptNavigator } from './gpt-problems';
+
+function TemplateMenu({
+  templates,
+  onClose,
+  onSelectTemplate,
+  selectedTemplate,
+}: {
+  templates: Template[];
+  onClose: () => void;
+  onSelectTemplate: (template: Template) => void;
+  selectedTemplate: Template;
+}) {
+  return (
+    <FixedPositionMenu
+      staticContent={
+        <Box display="flex" justifyContent="space-between">
+          <Typography
+            p="5px"
+            textAlign="center"
+            fontWeight="bold"
+            fontSize="20px"
+            color={PRIMARY_COL}
+          >
+            Template List
+          </Typography>
+          <IconButton sx={{ m: '2px' }} onClick={onClose}>
+            <CloseIcon />
+          </IconButton>
+        </Box>
+      }
+    >
+      <Box sx={{ cursor: 'pointer' }}>
+        {templates.map((template: Template) => (
+          <Typography
+            ml="20px"
+            mt="5px"
+            key={template.templateName}
+            onClick={() => onSelectTemplate(template)}
+            fontWeight={
+              template.templateName === selectedTemplate?.templateName
+                ? 'bold'
+                : undefined
+            }
+          >
+            {template.templateName}
+          </Typography>
+        ))}
+      </Box>
+    </FixedPositionMenu>
+  );
+}
+
+function TemplatePrompt({
+  prompt,
+  updateMessage,
+  updater,
+}: {
+  prompt: string;
+  updateMessage: string;
+  updater: string;
+}) {
+  const boxStyle = {
+    border: '1px solid gray',
+    borderRadius: '4px',
+    padding: '5px',
+    whiteSpace: 'pre-wrap' as const,
+    marginBottom: '5px',
+  };
+  return (
+    <>
```

Comment:
remove unnecessary fragment

Reply:
It is required because there is no single parent

Reply:
Ah...Okay. Didn't see the 2 children.